operator/internal/controller/auto/workspace_controller.go

```diff
@@ -419,9 +445,10 @@ func newStatefulSet(ctx context.Context, w *autov1alpha1.Workspace, source *sour
 									ContainerPort: WorkspaceGrpcPort,
 								},
 							},
-							Env:     w.Spec.Env,
-							EnvFrom: w.Spec.EnvFrom,
-							Command: command,
+							Env:        env,
+							EnvFrom:    w.Spec.EnvFrom,
+							Command:    command,
+							WorkingDir: "/share/workspace",
```

Comment:
Rationale: It is a good time to include a small usability improvement, of setting the working directory of the container to the Pulumi program workspace. This automatically sets the initial directory when you shell into a pod using vscode.